extensions/amp-form/0.1/amp-form.js

```diff
@@ -1442,13 +1470,34 @@ export class AmpForm {
   }
 }
 
+/**
+ * Returns all element who's form attribute is the `form`
+ * that match the selectors.
+ * These elements must be contained by the form or <input>s
+ * that are outside the form.
+ * @param {!HTMLFormElement} form
+ * @param {string} query
+ * @return {!Array<HTMLElement>}
+ */
+export function formElementsQuerySelectorAll(form, query) {
+  return Array.from(form.elements).filter((element) => {
```

Comment:
nit: implicit return. Including my other suggestion below:

```suggestion
  return Array.from(form.elements).filter((element) => matches(element, query));
```